src/FRP/Event/Time.purs

```diff
@@ -15,3 +19,38 @@ foreign import animationFrame :: Event Unit
 
 -- | Create an event which reports the current time in milliseconds since the epoch.
 foreign import withTime :: forall a. Event a -> Event { value :: a, time :: Number }
+
+-- | On each event, ignore subsequent events for a given number of milliseconds.
+debounce :: forall a. Number -> Event a -> Event a
+debounce period = debounceWith (map { period, value: _ })
+
+-- | Provided an input event and transformation, block the input event for the
+-- | duration of the specified period on each output.
```

Comment:
It might be nice to include a short example here, if there is one, and a simpler one above too.